mifospay/src/main/res/layout/fragment_history.xml

```diff
@@ -18,6 +18,18 @@
         android:layout_height="match_parent"
         android:scrollbars="vertical" />
 
+    <ImageView
```

Comment:
Can you please explain why you've been using an ImageView instead of a FloatingActionButton? 
As, there is Coordinator Layout being used and fabs are always more preferable in this case

Reply:
Tue only reason is we are not in AndroidX and apparently I am having an issue of setting background to the icon image. I have tried background, backgroundTint, src, etc.. If you have any way, do share :)

Reply:
Thanks for pointing this out.
No, i'm not proficient in markups and design attributes, so for that, I would've googled at my best if I had run in a similar scenario as yours.
I believe you too would've done so, therefore, on top of it, hereby, I'm marking this thread as _resolved_